### PR TITLE
Closes #134 - remove jQuery dependency from debug toolbar

### PR DESCRIPTION
**Pull request by mickaelandrieu**
* Remove jQuery code, added native javascript.
* Fixed some styles and positionning

### CODE REVIEW

File: Resources/scripts/Profiler/toolbar.html.twig [hunk 1]
```diff
@@ -100,20 +101,33 @@
...
 
 
-    $('#bb-profiler-close').click(function(){
-        $('#bb-profiler-panel').hide();
-    });
+    var panelLinks = document.querySelectorAll('#bb-profiler-panel .nav a');
+    for (i=0; i < panelLinks.length; i++) {
+        panelLinks[i].addEventListener('click', function completePanel() {
+            var hash = this.getAttribute('href');
+            console.log(hash);
```
Comment (eric-chau) [inline Resources/scripts/Profiler/toolbar.html.twig]: I think you forget to remove this ``console.log()`` =)

**Comment by ndufreche** [comment]
you can remove one 9 from z-index ?

**Comment by mickaelandrieu** [comment]
@ndufreche done.
ping @eric-chau

**Comment by ndufreche** [comment]
thanks

**Pull request merged**